weave/trace/weave_client.py

```diff
@@ -927,6 +925,107 @@ def get(self, ref: ObjectRef, *, objectify: bool = True) -> Any:
+        """
+        Retrieves all root Evaluation.evaluate calls in this project.
+
+        Returns:
+            CallsIter: An iterator over Call objects representing evaluation runs.
+                Supports slicing, iteration, and conversion to pandas DataFrames via `.to_pandas()`.
+
+        Examples:
+            Basic usage to get all evaluations:
+
+            >>> client = weave.init("my-project")
+            >>> evaluations = client.get_evaluation_calls()
+            >>> for eval_call in evaluations:
+            ...     print(f"Evaluation: {eval_call.id}")
+
+            Convert to a list:
+
+            >>> eval_list = list(client.get_evaluation_calls())
+
+            Convert to pandas DataFrame:
+
+            >>> eval_df = client.get_evaluation_calls().to_pandas(flatten=True)
+        """
+        evaluate_op_name = "Evaluation.evaluate"
+
+        kwargs = {
+            "query": {
+                "$expr": {
+                    "$contains": {
+                        "input": {"$getField": "op_name"},
+                        "substr": {"$literal": evaluate_op_name},
+                    }
+                }
+            }
+        }
+        if evaluation_uri:
+            kwargs["filter"] = CallsFilter(input_refs=[evaluation_uri])
+
+        return self.get_calls(**kwargs)
+
+    def get_score_calls(self, evaluation_uri: str | None = None) -> CallsIter:
+        """
+        Retrieves all score calls in this project.
+
+        Note:
+            Currently only works for imperative evaluations (using EvaluationLogger).
+
+        Returns:
+            CallsIter: An iterator over Call objects representing score calculations.
+                Supports slicing, iteration, and conversion to pandas DataFrames via `.to_pandas()`.
+
+        Examples:
+            Basic usage to get all score calls:
+
+            >>> client = weave.init("my-project")
+            >>> scores = client.get_score_calls()
+            >>> for score_call in scores:
+            ...     print(f"Score: {score_call.output}")
+
+            Convert to a list:
+
+            >>> score_list = list(client.get_score_calls())
+
+            Convert to pandas DataFrame for analysis:
+
+            >>> scores_df = client.get_score_calls().to_pandas(flatten=True)
+            >>> print(scores_df[['inputs', 'output', 'summary']])
+        """
+        score_json_path = "attributes._weave_eval_meta.score"
+
+        kwargs = {
+            "query": {
+                "$expr": {
+                    "$eq": [
+                        {
+                            "$convert": {
+                                "input": {"$getField": score_json_path},
+                                "to": "string",
+                            },
+                        },
+                        {"$literal": "true"},
+                    ]
+                }
+            },
+            "expand_columns": [score_json_path],
```

Comment:
this is unnecessary `attributes._weave_eval_meta.score` is accessible directly, without ref resolution